.github/workflows/ci_4337.yml

```diff
@@ -31,6 +31,18 @@ jobs:
       - run: |
           npm ci
           npm run test:e2e -w modules/4337
+  e2e-upstream-bundler:
```

Comment:
As suggested by OZ, can we move this to its own workflow file and make it run on a schedule?